lesson_17/dags/sql/transfer_customers_bronze_to_silver.sql

```diff
@@ -0,0 +1,51 @@
+DECLARE v_dag_logical_date DEFAULT CAST(CAST('{{ dag_run.logical_date }}' AS TIMESTAMP) AS DATE);
```

Comment:
I would create subfolders in SQL folder for each pipeline. For example `sql/sales/`, `sql/customers`